packages/dds/legacy-dds/src/array/sharedArray.ts

```diff
@@ -117,6 +120,21 @@ export class SharedArrayClass<T extends SerializableTypeForSharedArray>
 		return this.sharedArray.filter((item) => !item.isDeleted).map((entry) => entry.value);
 	}
 
+	/**
```

Comment:
toggle and toggleMove operations need internal entryIds so exposing them here. Since they are not added to ISharedArray interface I believe it is safe to go for this approach and access the entry Ids through SharedArrayClass

Reply:
Can the test code just hold onto the returned values?

Reply:
Can you clarify?

Reply:
The entryIds are emitted in events. The test could watch the events and keep the values to be used for toggle later.